functional-tests/test-bed/src/main/java/org/arquillian/smart/testing/ftest/testbed/project/BuildConfigurator.java

```diff
@@ -34,6 +34,8 @@
     private boolean mvnDebugOutput;
     private boolean enableSurefireRemoteDebugging = false;
     private boolean ignoreBuildFailure = false;
+    private boolean skipTests = false;
+    private String mavenLog = "false";
```

Comment:
This is rather fuzzy. I don't really see a reason to have it as a string / container for the log content and a "boolean flag".